pages/common/grunt.md

```diff
@@ -0,0 +1,35 @@
+# grunt
+
+> A JavaScript task runner for automating processes.
+
+- Run the default task process:
+
+`grunt`
+
+- Run one or more specific space-separated task(s):
+
+`grunt {{task_name}}`
+
+- Specify an alternative configuration file:
+
+`grunt --gruntfile {{path/to/file}}`
+
+- Specify an alternative base path for relative files:
+
+`grunt --base {{path/to/directory}}`
+
+- Specify an additional directory to scan for tasks in:
+
+`grunt --tasks {{path/to/directory}}`
+
+- Specify a local NPM package name to scan for tasks in:
```

Comment:
It's really minor, but on the npm website `npm` is written in lowercase.

Reply:
Not sure why I wrote it uppercase. :thinking: